### PR TITLE
Visit Finish Sending

### PR DESCRIPTION
**Pull request by Johngtka**
Features List:

- [X] `new Visit model`
- [X] `new Visit Service`
- [X] `creating new visit by sendVisit() method to mongodb`
- [X] `message when result is ok or not`

View:
| Part 1 | Part 2 |
| --- | --- |
| ezgif com-video-to-gif | ezgif com-video-to-gif(1) |

### CODE REVIEW

**Comment by Johngtka** [comment]
S&M commit proposal:
`feat: sending visit data to database (#37)`

File: src/app/services/visit.service.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+
+import { Observable } from 'rxjs';
+
+import { Visit } from '../models/visit';
+import { environment } from 'src/environments/environment.dev';
```
Comment (macrog) [inline src/app/services/visit.service.ts:7]: change import path from absolute to relative please

**Review by macrog — CHANGES_REQUESTED** [review]

**Comment by macrog** [comment]
checking test done ✔️ ✔️ 
| select 3 with note | select 3 deselect 1 with note | 
| --- | --- |
| image | image |

File: src/app/services/visit.service.ts (hunk 1 above)
Comment (Johngtka) [reply src/app/services/visit.service.ts:7]: the environment path ?

Comment (Johngtka) [reply src/app/services/visit.service.ts:7]: `import { environment } from '../../environments/environment.dev';` this?

**Review by macrog — APPROVED** [review]
good job!

**Pull request merged**